### PR TITLE
Adding permissions reference table

### PR DESCRIPTION
**Pull request by dse0**

### CODE REVIEW

**Review by nstokoe — APPROVED** [review]
Not a blocker but it would be nice to indicate which are model-scoped and which are project scoped.

**Comment by dse0** [comment]
> Not a blocker but it would be nice to indicate which are model-scoped and which are project scoped.

Good call, I mentioned it in `READ_DATA_MODEL`, I'll add the same note to `CREATE_` and `UPDATE_`, maybe can split it out to a separate column in a future update.

**Review by annanixon — APPROVED** [review]

File: pages/docs/users-and-permissions/permissions-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+| CREATE_PARTNER_PROJECT               | Can create and see metadata about Partner Projects                                                                                                                                                                                 |
+| CREATE_ROLE                          | Can create new Roles                                                                                                                                                                                                               |
+| CREATE_SAVED_VIEW                    | Can create new Saved Explorations                                                                                                                                                                                                  |
+| CREATE_USER                          | Can add new users to the Project, and can update global Project settings: Project access domains, Project invite URL, Slack integration, Default first day of week, Weekly granularity display                                     |
```
[the remark this answers is not in the document]
Comment (dse0) [reply pages/docs/users-and-permissions/permissions-reference.mdx:22]: Are there others missing from this list? Some project settings are *not* controlled by this, like the project name

File: pages/docs/users-and-permissions/permissions-reference.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| CREATE_COLLECTION                    | Can create new Collections                                                                                                                                                                                                         |
+| CREATE_DASHBOARD                     | Can create new Dashboards                                                                                                                                                                                                          |
+| CREATE_DATA_CONNECTION               | Can create new Data sources                                                                                                                                                                                                        |
+| CREATE_DATA_MODEL                    | Can create new Data Models. This permission allows for arbitrary SQL execution.                                                                                                                                                    |
```
[the remark this answers is not in the document]
Comment (dse0) [reply pages/docs/users-and-permissions/permissions-reference.mdx:13]: That would be `READ_SQL_RUNNER` further down. Do you mean this line might imply CREATE_DATA_MODEL gates the sql runner?

**Pull request merged**